packages/flex-plugin-e2e-tests/src/utils/browser.ts

```diff
@@ -86,6 +90,52 @@ export class Browser {
       });
   }
 
+  /**
+   * Prints all logs with level SEVERE
+   */
+  static async printLogs(): Promise<void> {
+    const logTypes = await this.getAvailableLogTypes();
+    const browserLogType = logging.Type.BROWSER;
+
+    if (logTypes.includes(browserLogType)) {
+      const logs = await this.browser.manage().logs().get(browserLogType);
+
+      if (logs.length) {
+        logger.info('Browser logs');
+        logs.forEach((entry) => {
```

Comment:
Only severes logs? why not warnings/errors? Also, use `filter` instead of `if` inside a `forEach`

Reply:
The available log levers are the following: https://source.chromium.org/chromium/chromium/src/+/master:chrome/test/chromedriver/logging.cc;l=74

Severe === Error. 
As for warnings - they usually just cause a lot of spam. In case of Flex, you will see a bunch of 'failed to start audio context' etc., which is not related to the failure and will just cause confusion for whoever is debugging.